resources/i18n/fr.json

```diff
@@ -390,7 +390,7 @@
                 "lastfmScrobbling": "Scrobbler vers Last.fm",
                 "listenBrainzScrobbling": "Scrobbler vers ListenBrainz",
                 "replaygain": "Mode ReplayGain",
-                "preAmp": "Pre-amplification ReplayGain (dB)",
+                "preAmp": "Pré-amplification ReplayGain (dB)",
```

Comment:
The "pré" prefix in French should be accentuated. [Source](https://www.larousse.fr/dictionnaires/francais/pr%C3%A9-/63268)